modules/volunteers/sitesupport/hourTable.js

```diff
@@ -110,6 +36,90 @@ export default {
     }
   },
   methods: {
+    load() {
+      this.records = null;
+      this.totalHours = 0;
+      this.totalSpentHours = 0;
+      this.hours = [];
+      this.totalVolunteers = null;
+
+      showSpinner();
+      this.summeryColumns = [
+        {value:'name', title:'Department Worked', source:null},
+
+        {value:'entry_count', title:'Total Entries', source:this.hours},
+        {value:'volunteer_count', title:'Total Volunteers', source:this.hours},
+        {value:'total_hours', title:'Total Hours', source: this.hours}
+      ];
+      this.detailColumns = [
+        {value:'name', title:'Department Worked', source:null},
+        {value:'end', title:'Ended', source: this.hours},
+        {value:'hours', title:'Hours', source: this.hours}
+      ];
+
+      apiRequest('GET', '/department','max_results=all')
+        .then((response) => {
+          const result = JSON.parse(response.responseText);
+          this.departments = [];
+          result.data.forEach((entry) => {
+            this.departments.push(entry);
+          });
+
+          this.departments.sort((a,b) => (a.name > b.name) ? 1 : -1);
```

Comment:
Would it be worth considering adjusting the query on the backend to do this sort for us?

Reply:
Either way, you said above that sorting is efficient in JavaScript so not sure if it is better to sort on the client or the server. I tend to like to push easy work to the client.

Reply:
Yeah, sorting on the client is probably fine. I have no strong preference here.